3_javascript/javascript1/week4/VoiceAssistant.js

```diff
@@ -0,0 +1,79 @@
+// Create a function called getReply(command). 
+//The function should return a response that corresponds to the command!
+
+// These are the commands you should be able to give the voice assistant:
+
+// Hello my name is Benjamin - Should save the name benjamin. and respond with "nice to meet you Benjamin". What if someone writes this twice?
+// What is my name - should respond with the name of the person.
+// What if the name has not yet been mentioned?
+// Add fishing to my todo - Should respond with "fishing added to your todo". 
+//Should add fishing to a list of todos
+// Add singing in the shower to my todo - Should add singing in the shower to a list of todos
+// Remove fishing from my todo - Should respond with "Removed fishing from your todo"
+// What is on my todo? - should respond with the todos. Fx you have 2 todos - 
+//fishing and singing in the shower
+// What day is it today? - Should respond with the date in a human readable format. 
+//E.g. if today is 30/8/2019 then it should respond with 30. of August 2019
+// Should be able to do simple math. fx what is 3 + 3 should respond with 6. Or what is 4 * 12 should respond with 48
+// Set a timer for 4 minutes - Should respond with "Timer set for 4 minutes". When 4 minutes is up: "Timer done". How do we set a timer in js? Google is your friend here!
+// Add one or more command to your voice assistant
+
+
+
+
+let userName = ""; // متغیر جهت ذخیره نام کاربر
+let todos = []; // لیست وظایف کاربر
+function getReply(command) {
+    const lowerCommand = command.toLowerCase(); // تبدیل دستور به حروف کوچک
```

Comment:
like how you are using toLowerCase , making it not sensitive ⭐

Reply:
I appreciate your mention  I think,what do you mean When you mention 'not sensitive,' do you mean that using toLowerCase allows the comparison to be case-insensitive? So, whether someone types in uppercase or lowercase, it won't affect the understanding or recognition of the input. If there's anything else you're want to mention about, feel free to ask!

Reply:
I meant that it is good practice to use it, you are making it in this way **not** sensitive, and if someone will decide to ask 'WhaT iS mY nAmE?' it will make the string look organised and your 'data' will be in the same format, in your case - in lower case 👍